src/portal/src/app/shared/components/global-search/search-result.component.ts

```diff
@@ -169,11 +176,27 @@ export class SearchResultComponent implements OnInit, OnDestroy {
                 this.onGoing = false;
                 this.originalCopy = searchResults; // Keep the original data
                 this.searchResults = this.clone(searchResults);
+                this.adjustOverlayTop();
             },
             error => {
                 this.onGoing = false;
                 this.msgHandler.handleError(error);
             }
         );
     }
+    getTopValue(): number {
+        const headerHeight: number = document.querySelector('navigator')?.clientHeight || 10;
+        const bannerHeight: number = document.querySelector('app-app-level-alerts')?.clientHeight || 10;
+        return headerHeight + bannerHeight;
+    }
+
+    private adjustOverlayTop(): void {
```

Comment:
if you use this function to re-render the dom, then you can remove the style binding in `[search-result.component.html](https://github.com/goharbor/harbor/pull/20794/files#diff-ec327ae356d30c4098c563bfce30fa1e746dfc33c449e9dcdd0a273c11297c7c)`

Reply:
Please remove this function